lib/src/hashtable.rs

```diff
@@ -14,12 +14,12 @@
 
 use crate::compress::MetablockData;
 use crate::constants::*;
+use bounded_utils::safe_x86_64;
 use bounded_utils::{
     bounded_u8_array, BoundedIterable, BoundedSlice, BoundedU32, BoundedU8, BoundedUsize,
 };
 use hugepage_buffer::BoxedHugePageArray;
-use safe_arch::safe_arch;
-use safe_arch::x86_64 as safe_x86_64;
+use safe_arch_macro::safe_arch;
```

Comment:
Can we remove this macro entirely? I believe it should always be a noop (can be a follow-up too)